src/diffusers/pipelines/controlnet/pipeline_controlnet.py

```diff
@@ -624,38 +626,14 @@ def check_image(self, image, prompt, prompt_embeds):
     def prepare_image(
         self,
         image,
-        width,
-        height,
         batch_size,
         num_images_per_prompt,
         device,
         dtype,
         do_classifier_free_guidance=False,
         guess_mode=False,
     ):
-        if not isinstance(image, torch.Tensor):
-            if isinstance(image, PIL.Image.Image):
-                image = [image]
-
-            if isinstance(image[0], PIL.Image.Image):
-                images = []
-
-                for image_ in image:
-                    image_ = image_.convert("RGB")
-                    image_ = image_.resize((width, height), resample=PIL_INTERPOLATION["lanczos"])
-                    image_ = np.array(image_)
-                    image_ = image_[None, :]
-                    images.append(image_)
-
-                image = images
-
-                image = np.concatenate(image, axis=0)
-                image = np.array(image).astype(np.float32) / 255.0
-                image = image.transpose(0, 3, 1, 2)
-                image = torch.from_numpy(image)
-            elif isinstance(image[0], torch.Tensor):
-                image = torch.cat(image, dim=0)
-
+        image = self.control_image_processor.preprocess(image).to(dtype=torch.float32)
```

Comment:
```suggestion
        image = self.control_image_processor.preprocess(image,  width=width, height=height).to(dtype=torch.float32)
```

shouldn't we pass width and height here?